src/NServiceBus.Core/Recoverability/Recoverability.cs

```diff
@@ -18,8 +18,11 @@ public Recoverability()
                 "Message recoverability is only relevant for endpoints receiving messages.");
             Defaults(settings =>
             {
-                settings.SetDefault(ImmedidateRetriesEnabled, true);
-                settings.SetDefault(DelayedRetriesEnabled, true);
+                settings.SetDefault(SlrNumberOfRetries, DefaultSecondLevelRetryPolicy.DefaultNumberOfRetries);
+                settings.SetDefault(SlrTimeIncrease, DefaultSecondLevelRetryPolicy.DefaultTimeIncrease);
+
+                settings.SetDefault(FlrNumberOfRetries, 5);
```

Comment:
It's the perfect spot but SLR defaults are in the `DefaultSecondLevelRetryPolicy` class and FLR defaults are inline. Maybe FLR defaults should also have their own container?

Maybe it this is one behavior now, both FLR and SLR defaults should simply be inlined here?